qutebrowser/keyinput/basekeyparser.py

```diff
@@ -366,6 +365,7 @@ def on_keyconfig_changed(self, mode):
 
     def clear_keystring(self):
         """Clear the currently entered key sequence."""
-        self._debug_log("discarding keystring '{}'.".format(self._keystring))
-        self._keystring = ''
-        self.keystring_updated.emit(self._keystring)
+        if self._keystring:
+            self._debug_log("discarding keystring '{}'.".format(self._keystring))
+            self._keystring = ''
+            self.keystring_updated.emit(self._keystring)
```

Comment:
Right - is https://github.com/The-Compiler/qutebrowser/pull/1843/commits/567fe0ec1006400f3b804f02a89d0caef8a8200e (which restores the previous unconditional behaviour) fine or should I check if the keystring gets actually changed?